product_replenishment_cost_default_seller/models/product_product.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, api
+from odoo.models import Model
+
+import odoo.addons.decimal_precision as dp
+
+
+class ProductProduct(Model):
+    _inherit = 'product.product'
+
+    @api.one
+    @api.depends('product_tmpl_id.standard_price', 'standard_price',
+                 'product_tmpl_id.seller_ids.price')
+    def _get_replenishment_cost(self):
+        if self.seller_ids:
+            self.replenishment_cost = self.seller_ids[0].price
+        else:
+            super(ProductProduct, self)._get_replenishment_cost()
+
+    replenishment_cost = fields.Float(
```

Comment:
why do you redefine this field ?

Reply:
Was thinking on old OpenERP versions.

Fixed.